app/src/main/res/layout/content_main.xml

```diff
@@ -44,7 +44,7 @@
           android:textColor="@android:color/white"
           app:backgroundTint="#1565c0"
           app:layout_constraintBottom_toBottomOf="parent"
-          app:layout_constraintEnd_toEndOf="parent"
+          app:layout_constraintRight_toRightOf="parent"
```

Comment:
@macgills But that's available from API 17. Or else I have to move it to Java and use ViewCompat instead of XML and I'm not a fan of moving the UI related stuff to Java.

Reply:
Ah, fair enough, maybe a comment then? I would always believe a right/left was a mistake

Reply:
@macgills Done.